Police-Data-Accessibility-Project/data-sources-app-v2
# Add logic for linking/unlinking data requests with locations; add `lo…

### Fixes

* https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/505

### Description

* Add logic for linking/unlinking data requests with locations
* Add `location_id` to `/typeahead/locations` logic

### Testing

* Run tests in `tests` directory 
* Inspect API to confirm expected appearance of affected endpoints

### Performance

* New endpoints bring with them separate performance overhead
* Addition of it to `/typeahead/location` logic marginally increases performance overhead
* Increased testing overhead

### Docs

* API documentation updated

### Breaking Changes

* No breaking changes.
